tableau_utilities/tableau_file/tableau_file.py

```diff
@@ -255,6 +262,11 @@ def enforce_column(self, column, folder_name=None, remote_name=None):
             elif not folder:
                 self.folders_common.folder.add(tfo.Folder(name=folder_name, folder_item=[folder_item]))
 
+            # Set display to show folders
+            self.layout.update(show_structure='true')
```

Comment:
@JustinGrilli  This code seems to be working except I'm getting stuck on altering the layout tags.  Can you tell me how to apply this change?